src/test/blockfilter_tests.cpp

```diff
@@ -67,6 +67,7 @@ BOOST_AUTO_TEST_CASE(blockfilter_basic_test)
     CMutableTransaction tx_2;
     tx_2.vout.emplace_back(300, included_scripts[2]);
     tx_2.vout.emplace_back(0, excluded_scripts[0]);
+    tx_2.vout.emplace_back(600, included_scripts[5]); // Script is empty
```

Comment:
So this should be `excluded_scripts[2]`?

nit, above in L62 add something along
```cpp
// This script is left empty.
// excluded_scripts[2];
```